src/Utils/Utils.php

```diff
@@ -0,0 +1,75 @@
+<?php
+namespace WPGraphQL\Utils;
+
+class Utils {
+
+	/**
+	 * Maps new input query args and sanitizes the input
+	 *
+	 * @param array $args The raw query args from the GraphQL query
+	 * @param array $map  The mapping of where each of the args should go
+	 *
+	 * @since  0.5.0
+	 * @return array
+	 * @access public
+	 */
+	public static function map_input( $args, $map ) {
```

Comment:
I think we should approach this a little differently where we recursively call sanitization methods so we can sanitize multiple levels of input. Currently, this only supports a single level of input, and only sanitizes strings. Another issue is that we're doing the type checking based on the incoming value rather than config. So if an input is expecting an integer, but a client sends a number typeset as a string, it will fail. Instead we could look up what type the arg is _supposed_ to be from config, and then sanitize the value based off that.

Reply:
Probably worth a different issue for addressing this. This function itself didn't change here, just `phpcbf` formatting, I believe